pkg/webhook/workspace/config.go

```diff
@@ -117,7 +129,28 @@ func Configure(ctx context.Context) error {
 		log.Info("Created workspace validating webhook configuration")
 	}
 
-	server.GetWebhookServer().Register(validateWebhookPath, &webhook.Admission{Handler: NewResourcesValidator()})
+	server.GetWebhookServer().Register(validateWebhookPath, &webhook.Admission{Handler: NewResourcesValidator(saUID)})
 
 	return nil
 }
+
+func controllerUID(ctx context.Context, c client.Client) (string, error) {
```

Comment:
```suggestion
func controllerSAUID(ctx context.Context, c client.Client) (string, error) {
```

Reply:
Does it make sense to move it somewhere like `/internal/controller/service_account.go` ?

Reply:
For now, it's only used for webhooks, but we could always move it to an internal utility if we find we need the UID elsewhere.